.github/workflows/fronts.yml

```diff
@@ -0,0 +1,34 @@
+name: DCR Fronts Ophan tag status
+on:
+  # Triggers the workflow on push and pull request events but only for the main branch
+  push:
+    branches: [main]
+
+  # Allows you to run this workflow manually from the Actions tab.
+  workflow_dispatch:
+
+jobs:
+  data-link-name:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Setup deno
+        uses: denolib/setup-deno@v2
+        with:
+          deno-version: v1.21.0
+
+      - name: Run Deno script
+        id: text
```

Comment:
What's this doing?

Reply:
My understanding is that it allows us to use the output of the script as a variable. Might be redundant with `"::set-output text=$TEXT`.